src/main/kotlin/org/rust/lang/RustSdkType.kt

```diff
@@ -24,9 +24,15 @@ class RustSdkType : SdkType("Rust SDK") {
     override fun suggestHomePath(): String? {
         if (SystemInfo.isUnix) {
             // Check whether there is multi-rust installation
-            val multi = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
-            if (multi.exists())
-                return multi.absolutePath
+            val root = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
+            if (root.exists()) {
+                listOf("stable", "beta", "nightly").forEach {
+                    val multiSub = File(root, it)
+                    if (multiSub.exists()) {
+                        return multiSub.absolutePath
+                    }
+                }
```

Comment:
Yes, this is intentional, I think it is a good default behavior. 

* selecting just `/toolchains` is bad, because it is not a valid Rust home. It is selected as default, but when the user clicks next, he sees an error message. It was confusing for me and that's why I created the issue in the first place ;) 
* stable and nightly are the common case
* if user has only particular versions, `tryFindSDKHomeInPATH` may provide a better guess.